### PR TITLE
[tangoinfo] Improve description.

### PR DESCRIPTION
**Pull request by Sophist-UK**

### CODE REVIEW

File: plugins/tangoinfo/tangoinfo.py [hunk 1]
```diff
@@ -2,13 +2,13 @@
 PLUGIN_NAME = "Tango.info Adapter"
 PLUGIN_AUTHOR = "Felix Elsner, Sambhav Kothari, Philipp Wolfer"
 PLUGIN_DESCRIPTION = """
-<p>Load genre, date and vocalist tags from the online database
-<a href"http://tango.info">tango.info</a>.</p>
+<p>Load genre, date and vocalist tags for latin dance music 
+from the <a href"http://tango.info">tango.info</a>.</p>
```
Comment (phw) [inline plugins/tangoinfo/tangoinfo.py]: To me "from the http://..." sounds strange. Would this really be said like this or rather just "from http://...."?

**Review by phw — COMMENTED** [review]
I wonder if this plugin still works or if https://github.com/metabrainz/picard-plugins/issues/179 is still an issue.

**Comment by Sophist-UK** [comment]
> I wonder if this plugin still works or if #179 is still an issue.

No idea - I never use it - just have time to restart my Picard usage and fixing things as I find them when recreating my settings and scripts.

**Review by phw — APPROVED** [review]

**Comment by phw** [comment]
> No idea - I never use it

Same for me. Found it difficult to test last time I tried, mostly because I was searching for a test case. Anyway, let's merge the description improvements for now :D

**Pull request merged**